.npmignore

```diff
@@ -0,0 +1 @@
+/example
```

Comment:
Here we probably need more. See https://github.com/Wiredcraft/file-register/blob/master/.npmignore as an example (don't simply copy).